src/test/java/jalau/at18/katas/likes/MainTest.java

```diff
@@ -0,0 +1,15 @@
+package jalau.at18.katas.likes;
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+import jalau.at18.katas.likes.rodrigov.Main;
+public class MainTest {
+    @Test
+    public void shouldReturnNamesGaveLike() {
```

Comment:
Please also add tests or asserts for all cases mentioned in the kata requirement
```
[]                                -->  "no one likes this"
["Peter"]                         -->  "Peter likes this"
["Jacob", "Alex"]                 -->  "Jacob and Alex like this"
["Max", "John", "Mark"]           -->  "Max, John and Mark like this"
["Alex", "Jacob", "Mark", "Max"]  -->  "Alex, Jacob and 2 others like this"
```

Also, there is a slight diff between the requirement and the implementation you did, requirement expectes to send an array as an input instead of a string. Please update code and tests accordingly

Reply:
I added all the tests to /likes/MainTest.java